pkg/bootstrap/docker/dockerhelper/helper.go

```diff
@@ -118,31 +118,11 @@ func (h *Helper) CheckAndPull(image string, out io.Writer) error {
 	}
 	glog.V(5).Infof("Image %q not found. Pulling", image)
 	fmt.Fprintf(out, "Pulling image %s\n", image)
```

Comment:
The output stream in cluster up is doing indenting for me. I also don't want the output to be log-like output since it's just a regular cli cmd. Only if you turn up the loglevel you get the glog output.